Source/WebCore/svg/SVGComponentTransferFunctionElement.cpp

```diff
@@ -92,7 +92,7 @@ void SVGComponentTransferFunctionElement::svgAttributeChanged(const QualifiedNam
 {
     if (PropertyRegistry::isKnownAttribute(attrName)) {
         InstanceInvalidationGuard guard(*this);
-        SVGFilterPrimitiveStandardAttributes::invalidateFilterPrimitiveParent(this);
+        SVGFilterPrimitiveStandardAttributes::markFilterEffectForRebuildOnParent(*this);
```

Comment:
It's not as straightforward to do like SVGxxxLightingElement because the existing mechanism to cause updates to the effect is to go through primitiveAttributeChanged, and that function doesn't identify the element with the changed attribute (because it's assumed to be the primitive element itself, and the lighting filter primitive elements only care about a single child element, so it kind of works to call primitiveAttributeChanged).

But I guess we could add second versions of primitiveAttributeChanged / setFilterEffectAttribute that take the child element?